### ISSUE TITLE
Missing Tab id causes broken ARIA reference

### ISSUE THREAD

**Issue by vga-** [react-bootstrap/react-bootstrap#1854]
Hi All,

According to the documentation, the id prop of the Tabs component is required for a11y. Using this id, the tab pane and the tab should be given corresponding ids and aria attributes. For example:

```javascript
<Tabs id="example">
     <Tab>content<Tab>
<Tabs>
```

should generate something like:

```html
<div id="exmaple">
    <ul ...>
        <li role=...>
            <a id="example-tab-1" aria-controls="example-pane-1" ...>link text</a>
        </li>
    </ul>
    <div class="tab-content">
        <div id="example-pane-1" aria-labelledby="example-tab-1">
            content
        </div>
    </div>
</div>
```

I believe this is the intended behavior. However, **the `<a>` id is not being generated**. This is an a11y error, there should never be an aria-labelledby without a corresponding element with that id.

I've tried to add an id to the tab myself, but I get the following error:

> In the context of a TabContainer, TabPanes are given generated `id` and `aria-labelledby` attributes for the sake of proper component accessibility. Any provided ones will be ignored. To control these attributes directly provide a `generateChildId` prop to the parent TabContainer.

I've also tried setting the id and the aria-labelledby prop of the Tab and I get the same error. Removing the id from the Tabs component doesn't solve the problem either.

I'm pretty sure I can spin up my own tabs using the lower level TabContainer, TabContent, and TabPanel components, but I'd rather try to fix the issue for the easier to use components.

I'm going to look into the code and hopefully submit a pull request. In the mean time can someone confirm that my diagnosis is correct...the tabs in the documentation exhibit the missing <a> id and broken ARIA reference behavior. There should be an id in the `<a>`, correct?

Thanks!

**Comment by jquense** [comment]
confirmed, its a regression caused by changing how we pass `id` to navitems: https://github.com/intljusticemission/casenator/blob/r-aft/Casenator.Web/Scripts/app/components/filters/Filterable.js#L136

linkId doesn't exist anymore since props are passed directly to the anchor now, but we didn't refactor it in the link above

**Comment by vga-** [comment]
Link is broken. I'd like to help on this when I get a better understanding of the issue.

**Comment by jquense** [comment]
sorry copy paste error, updated

**Comment by taion** [comment]
This got fixed
